### PR TITLE
feature(Stage mode): Joining fixed

### PR DESCRIPTION
**Pull request by PAKovalev**
Joining space was fixed

### CODE REVIEW

File: packages/app/src/stores/MainStore/models/AgoraStore/AgoraStore.ts [hunk 1]
```diff
@@ -36,14 +36,10 @@ const AgoraStore = types
...
-          api.spaceIntegrationsRepository.fetchStageModeStatus,
-          {spaceId: spaceId ?? self.spaceId}
-        );
-      }
-
-      return undefined;
+      return yield self.spaceIntegrationsRequest.send(
+        api.spaceIntegrationsRepository.fetchStageModeStatus,
+        {spaceId: spaceId || self.spaceId || ''}
+      );
```
Comment (Daboo1998) [inline packages/app/src/stores/MainStore/models/AgoraStore/AgoraStore.ts:42]:
```suggestion

      if (!spaceId && !self.spaceId) {
        return undefined
      }
       
      return yield self.spaceIntegrationsRequest.send(
        api.spaceIntegrationsRepository.fetchStageModeStatus,
        {spaceId: spaceId || self.spaceId}
      );
```

**Review by Daboo1998 — CHANGES_REQUESTED** [review]

File: packages/app/src/stores/MainStore/models/AgoraStore/AgoraStore.ts (hunk 1 above)
Comment (PAKovalev) [reply packages/app/src/stores/MainStore/models/AgoraStore/AgoraStore.ts:42]: Why we have two spaceId at all?

Comment (PAKovalev) [reply packages/app/src/stores/MainStore/models/AgoraStore/AgoraStore.ts:42]: Eslint error will appear.

Comment (PAKovalev) [reply packages/app/src/stores/MainStore/models/AgoraStore/AgoraStore.ts:42]: We should refactor this. Not for now.

Comment (Daboo1998) [reply packages/app/src/stores/MainStore/models/AgoraStore/AgoraStore.ts:42]: I think for rejoining you have spaceId undefined and then you get spaceId that was defined before. If you join for first time, the spaceId gets defined

Comment (Daboo1998) [reply packages/app/src/stores/MainStore/models/AgoraStore/AgoraStore.ts:42]: But ok lets leave it, hope it won't produce bugs

**Review by Daboo1998 — APPROVED** [review]

**Review by aminbksi — APPROVED** [review]

**Pull request merged**